source/extensions/filters/http/aws_lambda/request_response.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+// The structures are used for the purpose of JSON (de)serialization.
+package source.extensions.filters.http.aws_lambda;
+
+import "validate/validate.proto";
+
+message Request {
```

Comment:
Might want to also reference [envoy's auth service api](https://github.com/envoyproxy/envoy/blob/e1962d76c073bb48c57acd0ff2b57390a22394b7/api/envoy/service/auth/v2/attribute_context.proto) since it also has mappings for HTTP requests in protos.

I'm not saying yours is worse or better.

Reply:
good tip 👍